ona/authentication.py

```diff
@@ -39,7 +40,8 @@ def get_api_token(json_web_token):
     # globally mainly because there isn't a test for it
     try:
         jwt_payload = decode_payload(json_web_token)
-        api_token = get_object_or_404(Token, key=jwt_payload.get('api-token'))
+        api_token = Token.objects.using("kobocat").select_related(
```

Comment:
What happens if the Token doesn't exist ?

Reply:
I'm fine with this triggering a 404. I think the token should be generated in Ona

Reply:
Same but it seems currently the `Token.DoesNotExist error` isn't handled here ?